open-hackathon-adminUI/src/app/views.py

```diff
@@ -6,9 +6,21 @@
 from flask_admin import BaseView, expose, Admin, AdminIndexView
 from decorators import role_required
 from constants import ROLE
+from functions import get_config , delete_remote, get_remote, safe_get_config
+import json
 
+###########
+from flask_login import login_required, current_user, logout_user, login_user
+from admin.login import login_providers, LoginUser
+from flask import Response, render_template, abort, request, session, g, redirect, make_response
 
-class HomeView(AdminIndexView):
+
+
+
+
+
+
```

Comment:
why add so many blank lines? Please make sure the status of each py file is green